kolibri/core/assets/src/state/actions.js

```diff
@@ -223,6 +223,36 @@ function getCurrentSession(store) {
   .catch(error => { handleApiError(store, error); });
 }
 
+function getFacilityConfig(store) {
+  const coreApp = require('kolibri');
+  const FacilityCollection = coreApp.resources.FacilityResource
+    .getCollection()
+    .fetch();
+
+  return FacilityCollection.then(facilities => {
+    const currentFacilityId = facilities[0].id;
+    const facilityConfigCollection = coreApp.resources.FacilityDatasetResource
+      .getCollection({ facility_id: currentFacilityId })
+      .fetch();
+    return facilityConfigCollection
```

Comment:
When working on the config UI, I wrote a helper that converts between the python snake_case and JS camelCase. Maybe you could import it from here

http://github.com/jonboiser/kolibri/blob/9c65ee35b99a82d5d1e582d87c832e306b9ffa9a/kolibri/plugins/management/assets/src/state/facilityConfigPageActions.js#L20

And the conversion would be like `convertFacilityDataset(facilityConfig[0], true /* formatForClient param */`.